src/components/timeline/timeline-event/timeline-event.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, ElementRef, Renderer2 } from '@angular/core';
+
+@Component({
+    selector: 'ux-timeline-event',
+    templateUrl: './timeline-event.component.html'
+})
+export class TimelineEventComponent {
+
+    @Input() badgeColor: string;
+    @Input() badgeTitle: string;
+    @Input() title: string;
+    @Input() timestamp: Date;
+    
+    constructor (
+        private _elementRef: ElementRef,
+        private _renderer: Renderer2,
+    ) {}
+
+    ngAfterViewInit(): void {
+        this._renderer.setAttribute(this._elementRef.nativeElement, 'side', 'right');

```

Comment:
Is this class needed on the timeline event too? If it is use the host property on the decorator as well

Reply:
The 'side' attribute is no longer used.